src/CarbonAware.WebApi/src/Models/CarbonIntensityBatchDTO.cs

```diff
@@ -0,0 +1,9 @@
+namespace CarbonAware.WebApi.Models;
+
+using System.Text.Json.Serialization;
+
+[Serializable]
+public record CarbonIntensityBatchDTO : CarbonIntensityBaseDTO { }
```

Comment:
hmm, interesting. Any reason why this inheritance is without inheritance? Don't see the value added.

Reply:
Yes, see @bderusha

Reply:
@bderusha

Reply:
The value here is in signaling to future developers that the request object and the response object here are tightly coupled.  This pattern gives future developers a clear home for response only attributes, request only attributes, and shared attributes.

There are other patterns that could be used effectively here as well, but I tend to like this one.   Either way I don't think it should block this PR.  We can certainly have a wider discussion the merits and tradeoffs before the implementation of this spec merges.

Reply:
We will merge but bring this up for further discussion